src/components/presentational/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,139 @@
+import { format } from 'date-fns'
+import { LayoutContext } from '..'
+import { CartesianGrid, Line, LineChart as ReChartsLineChart, ReferenceLine, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts'
+import "./LineChart.css"
+import { ColorDefinition, resolveColor } from '../../../helpers/colors'
+import { useContext } from 'react'
+import React from 'react'
+
+export type LineChartPreviewState = "Default";
+
+export interface LineChartThreshold {
+    value: number
+    referenceLineColor?: ColorDefinition
+    overThresholdLineColor?: ColorDefinition
+}
+
+export interface LineChartDataPoint {
+    value: number,
+    date: Date
+}
+
+export interface LineChartAxis {
+    yRange?: number[]
+    yIncrement: number;
+    xRange?: number[];
+    xIncrement: number;
+}
+
+export interface LineChartProps {
+    data: LineChartDataPoint[],
+    axis?: LineChartAxis,
+    lineColor?: ColorDefinition,
+    thresholds?: LineChartThreshold[],
+    innerRef?: React.Ref<HTMLDivElement>
+}
+
+export default function MdhLineChart(props: LineChartProps) {
```

Comment:
I definitely wouldnt' call this "LineChart" as that is far too vague of a name.  I think we should make this at least somewhat opininated.  

Putting aside whether it makes sense to combine this with @ajmenca 's PR, I would propose that this be called "IntradayLineChart" and be hardcoded to a 24 hour interval with the x-axis as I described and it could take data as data points in that interval...do not think it makes sense to make a generally reusable "line chart" as that is what recharts already gives us

Reply:
I do agree LineChart is too vague. 
What I do want to avoid is recreating the threshold interface and the display of it across components. We currently consume BarChartThreshold in vbuilder. Adding LineChartThreshold here is no help. Would be nice to have a simple ChartThreshold interface that is used across charts, along with not having to copy/paste the display of thresholds across components.

Reply:
I will integrate the feedback around making it more opinionated and hang tight on the Threshold stuff